mypy/stubgen.py

```diff
@@ -596,6 +596,9 @@ def process_namedtuple(self, lvalue: NameExpr, rvalue: CallExpr) -> None:
         elif isinstance(rvalue.args[1], ListExpr):
             list_items = cast(List[StrExpr], rvalue.args[1].items)
             items = '[%s]' % ', '.join(repr(item.value) for item in list_items)
+        elif isinstance(rvalue.args[1], TupleExpr):
+            list_items = cast(List[StrExpr], rvalue.args[1].items)
+            items = '(%s)' % ', '.join(repr(item.value) for item in list_items)
```

Comment:
This would actually be broken if the original tuple had only one item. Reusing the previous stanza would avoid that.

Reply:
@gvanrossum oh, now I see - without the trailing comma, it's a generator expression...